### ISSUE TITLE
[Keyboard Navigation - Soft Min Max]: 'Soft Min Max' button does not receive keyboard focus making it inaccessible by keyboard.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66246]
### User Experience: 
Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls If interactive controls present on any page or pane are not keyboard accessible, hence fail to use their functionality.  Note: User credentials should NOT be included in the bug.

Repro Steps:
Open #Url: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcase pane. Select any one of the data such as ,"Time Series Graphs". "Visualizations" page opens, Navigate to the "Soft Min Max" pane and expand it. Verify the issue.

### Actual Result: 
The 'Soft Min Max' button is not accessible through keyboard as keyboard focus does not move to it.

### Expected Result:
 Keyboard focus should move to 'Soft Min Max' button, and it should be accessible.
Note:  The same issue exists throughout the application where this type of control appears.

### Severity
`SEV2`

**Comment by khushijain21** [comment]
Hello @natellium, I can access the soft min/max button via the keyboard as you can see in this picture. Let me know if I'm getting this right. Thanks!
![image](https://github.com/grafana/grafana/assets/57278642/86b5a5a3-fadf-4897-8a9f-ceba88a722bd)

**Comment by evictorero** [comment]
Thanks @khushijain21 for taking the time! Closing it since It's working as expected.
